### ISSUE TITLE
PHP 8.1 update

### ISSUE THREAD

**Issue by sl0wik** [GoogleCloudPlatform/php-docker#554]
PHP 8.1 is now available in "Public Preview" for App Engine standard (https://issuetracker.google.com/issues/170766508#comment76).

It would be amazing if this repo could catch up.

**Comment by frost-byte** [comment]
I'm just going to operate under the presumption that this platform is going to die, there's no where to see any kind of long term action plan. I haven't found any indication that they're actively working on extending its life.

**Comment by sl0wik** [comment]
From my research, it looks like this repo is dead because Google is migrating to buildpacks (ref.: https://github.com/GoogleCloudPlatform/buildpacks/).

**Comment by frost-byte** [comment]
You may be right, but they currently aren't listing php as a runtime on the repo you listed.

Actually I do see it mentioned at one point, but there are several prominent places where it's not mentioned with the other runtimes.

**Comment by sl0wik** [comment]
It's there - I'm already using it.

Example:
```
pack build $imageName --builder=gcr.io/gae-runtimes/buildpacks/php81/builder:latest --env=X_GOOGLE_TARGET_PLATFORM=gae
```

You can dockerize `pack` as well so you don't need `pack` installed locally.

Its perfect for production builds, but to use it locally some things need to be figured (check https://github.com/GoogleCloudPlatform/buildpacks/issues/218).

**Comment by html5maker** [comment]
@sl0wik have you managed to modify the nginx using buildpacks to serve static files like jpg, png?

**Comment by sl0wik** [comment]
No. I'm using it mainly for APIs. Track https://github.com/GoogleCloudPlatform/buildpacks/issues/234
